src/EnergyPlus/CMakeLists.txt

```diff
@@ -570,7 +570,12 @@ install( TARGETS energyplusapi
   LIBRARY DESTINATION ./ 
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX AND ENABLE_LTO)
+if (MINGW)
```

Comment:
@lefticus @nealkruis I've changed the logic here to keep the old GCC behavior (no link between energypluslib and energypluslib2) and only do the link for MinGW. If there are no objections, I'll merge this later today.

Also @Myoldmopar @kbenne